nox/tasks.py

```diff
@@ -166,7 +169,9 @@ def honor_list_request(
         return manifest
 
     # If the user just asked for a list of sessions, print that
-    # and be done.
+    # and any docstring specified in noxfile.py and be done.
+    if manifest.module_docstring:
+        print(manifest.module_docstring)
```

Comment:
We need to deal with whitespace here, which is different depending on the various docstring styles:

```suggestion
        print(manifest.module_docstring.strip(), end="\n\n")
```

This should handle the following styles:

```python
# we need to ensure a blank line after the docstring
"""Automation using Nox."""

# ...while avoiding a double blank line here:
"""Automation using Nox.
"""

# and this should not start with an initial blank line:
"""
Automation using Nox.
"""
```

Reply:
Good spot! I only ever use the third docstring style so this didn't occur to me.

I've incorporated your suggested changes now.